frontend/package.json

```diff
@@ -43,7 +43,7 @@
     "transformIgnorePatterns": [
       "<rootDir>/node_modules/(?!(lodash-es|@console|@novnc|@spice-project)/.*)"
     ],
-    "testRegex": "/__tests__/.*\\.spec\\.(ts|tsx|js|jsx)$",
+    "testRegex": "/__tests__/.*\\.(ts|tsx|js|jsx)$",
```

Comment:
This can't be right. We need `spec` in there because we put non-test files inside the `__tests__` dir as support utils or data.

Reply:
Exactly, we need this for all the monorepo tests, otherwise it blows up on test helpers (and rightly so).

```sh
$ yarn test packages
..
packages/dev-console/src/components/pipelines/__tests__/pipeline-mock.ts
..
Your test suite must contain at least one test.
```

We could use Jest [projects](https://jestjs.io/docs/en/configuration.html#projects-array-string-projectconfig) feature, or just git-mv the 8 files in `frontend/__tests__` to correctly follow the `spec` convention.

Reply:
@vojtechszocs just make `testRegex` an array. One for `packages` and one for everything else.

Reply:
@vojtechszocs actually just move the files to the correct format and be done with it. It's where we want to go anyways.